### PR TITLE
Feature#99

### PR DESCRIPTION
**Pull request by ghost**
Update Product
View Product

### CODE REVIEW

File: resources/views/product/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <input type="number" name="price" class="form-control" value="{{$product->price}}">
+
+        <label>{{trans('product.description')}}</label>
+        <input name="description" class="form-control" value="{{$product->description}}">
+
+        <label>{{trans('product.quantity')}}</label>
+        <input type="number" name="quantity" class="form-control" value="{{$product->quantity}}">
+
+        <label>{{trans('product.category')}}</label>
+        {!! Form::select('category_id', ['1' => 'Fashion', '2' => 'Electric'], $product->category_id, ['class'=>'form-control']) !!}
```
Comment (dzungnguyenh) [inline resources/views/product/show.blade.php:21]: [rank A] why did you hard code categories here?

**Review by dzungnguyenh — CHANGES_REQUESTED** [review]
please check my comment

File: resources/views/product/show.blade.php (hunk 1 above)
Comment (ghost) [reply resources/views/product/show.blade.php:21]: I wait category issue finish

**Review by dzungnguyenh — APPROVED** [review]
OK. Please remember to update categories list.

**Pull request merged**